cmd/github-ldap-user-group-creator/main.go

```diff
@@ -42,6 +43,7 @@ func parseOptions() *options {
 	fs.StringVar(&opts.logLevel, "log-level", "info", fmt.Sprintf("Log level is one of %v.", logrus.AllLevels))
 	fs.BoolVar(&opts.dryRun, "dry-run", true, "Whether to run the controller-manager with dry-run")
 	fs.StringVar(&opts.mappingFile, "mapping-file", "", "File to the mapping results of m(github_login)=kerberos_id.")
+	fs.StringVar(&opts.groupsFile, "groups-file", "", "The file to store the groups")
```

Comment:
Is this an output of the tool? An input? What is the format of the file?

Reply:
Updated.